Changes

```diff
@@ -1,8 +1,10 @@
 {{$NEXT}}
 
-1.0.4     2021-01-26 08:00:51+00:00 UTC
+1.0.5 2021-11-12 
```

Comment:
The cpan release will automatically add version and datetime
https://wikijs.deriv.cloud/en/Backend/guides/perl/cpan/cpan-maintenance#updating-the-changes-log

```suggestion
{{$NEXT}}
	- Added alias names for Eswatini, Trinidad & Tobago, Côte D'Ivoire
```